src/d3a/d3a_core/simulation.py

```diff
@@ -377,7 +377,7 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
                 bid_offer_matcher.event_tick(
                     is_it_time_for_external_tick=global_objects.external_global_stats.
                     is_it_time_for_external_tick(current_tick_in_slot),
-                    slot_completion=f"{int(self.progress_info.percentage_completed)}%",
+                    slot_completion=f"{int((tick_no/config.ticks_per_slot) * 100)}%",
```

Comment:
Can you please explain, why the `self.progress_info.percentage_completed` can not be used any more? Are the values wrong?

Reply:
This is the percentage completed compared to the whole simulation, what Chris asked for is the percentage completed in a market cycle

Reply:
I see, thanks a lot!